hasher-matcher-actioner/webapp/src/Api.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+// TODO figure out root path. For now copy your own URL gateway here
+const API_ROOT = 'https://<APPID>.execute-api.us-east-1.amazonaws.com'
```

Comment:
(Question for Brian maybe): Any way to force this to require someone to type in something here?